app/controllers/collections_controller.rb

```diff
@@ -53,6 +53,25 @@ def update
         end
       end
       @collection.update_attributes({:funder => newfunder})
+      @collection[:identifier].each do |id|
+        if /\Adoi:10.5072\/FK2/ =~ id
```

Comment:
Something like this?

# if this dataset has an existing identifier which is a DOI assigned by VT libraries through EZID;
# that is, it is started with the unique prefix associated with VT libraries 
if /\A#{Ezid::Client.config.default_shoulder}/ =~ id